src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,150 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def configoptions():
+    "Manages Sentry options."
+
+
+@configoptions.command()
+@click.option("--key", "-k", required=True, help="The key of the option to fetch.")
+@configuration
+def fetch(key):
+    "Fetches the option for the given key."
+    click.echo(get(key))
+
+
+@configoptions.command()
+@configuration
+def list():
+    "Fetches all options."
+    from sentry import options
+
+    for opt in options.all():
+        # click.echo(f"{opt.name} ({opt.type}) = {options.get(opt.name)}")
+        click.echo(f"{opt.name}: {options.get(opt.name)}")
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def patch(filename, dryrun):
+    "Updates, gets, and deletes options that are each subsectioned in the given file."
+    import yaml
+
+    if dryrun:
+        click.echo("Dryrun flag on. ")
+    with open(filename) as stream:
+        configmap_data = yaml.safe_load(stream)
+        data = configmap_data.get("data", {}).get("options-patch.yaml", "")
+
+        keysToFetch = data.get("fetch", {})
+        keysToFetch = (line for line in keysToFetch.split("\n") if line)
```

Comment:
Why do you need to split keyToUpdate by new line ? When you loaded the yaml, structured options would have become separate keys in a dictionary.

Reply:
The key value pairs in the dictionary are represented as a whole string. Originally I implemented them as a dictionary that gets read into a python dict, but Kyle mentioned that some of the key value pairs might not be that easy (like a seralized json object) so treating everything as a string might be better.

Example:

```yaml
options-strict.yaml: |
        key: "value"
        key: "value"
        key: "value"
```

Reply:
This is a reasonable point. Neo and I spoke about it at length. I think this is something that will change when we start wiring it up to the k8s job and start seeing how the configmap gets mounted into the file system.

Reply:
> but Kyle mentioned that some of the key value pairs might not be that easy (like a seralized json object) so treating everything as a string might be better.

We have constraint in what is an option today: simple values, lists, sets, mapping. Everything is supposed to be simply serializable so we are not supposed to have this problem. 

A few arguments against going first with strings: 
- escaping. You will have to escape values to make them valid json/yaml when they are in a string. This will make mistakes easier.
- changing back to raw yaml would become more complex

Reply:
Yeah, totally reasonable. I'm going to let the yaml library to my validation, etc. for me. I'll change this before we merge this PR.